src/nxdoc/web-ui/web-ui-release-notes/web-ui-release-notes-3-0-37.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Version 3.0.37
+description: Discover what's new in Web UI 3.0.37.
+review:
+  comment: ''
+  date: '2024-10-16'
+  status: ok
+toc: true
+labels:
+tree_item_index: 968
+hidden: true
+---
+
+{{{multiexcerpt 'matching-notes' page='web-ui-release-notes'}}}
+
+{{! multiexcerpt name='web-ui-updates'}}
+
+## What’s New in Web UI for LTS 2021 (Version 3.0.37)
+
+- Integrated github action to schedule Veracode scan.<br/>[[WEBUI-1572](https://jira.nuxeo.com/browse/WEBUI-1572)]
+
+- Updated upload artifact package version from v2 to v3.<br/>[[WEBUI-1578](https://jira.nuxeo.com/browse/WEBUI-1578)]
+
+- null check added in _dataChanged function so that it will not throw error in console for nuxeo-selectivity.<br/>[[ELEMENTS-1657](https://jira.nuxeo.com/browse/ELEMENTS-1657)]
```

Comment:
Can we replace "null" with Null? I would suggest this statement:
"Null check added in _dataChanged function so that it does not throw error in console for nuxeo-selectivity."